package.json

```diff
@@ -67,9 +70,9 @@
     "metro-react-native-babel-preset": "0.58.0",
     "prettier": "1.19.1",
     "react": "16.9.0",
-    "react-native": "0.61.5",
+    "react-native": "0.63.4",
     "react-native-gesture-handler": "1.6.0",
     "react-native-svg": "11.0.1",
-    "typescript": "3.8.3"
+    "typescript": "4.2.2"
```

Comment:
Some of these dependencies could probably be moved into `peerDependencies` at some point in the future too.